neuror/sanitize.py

```diff
@@ -100,7 +100,8 @@ def sanitize_all(input_folder, output_folder, nprocesses=1):
     if nprocesses == 1:
         results = map(func, morphologies)
     else:
-        results = Pool(nprocesses).imap_unordered(func, morphologies, chunksize=100)
+        with Pool(nprocesses) as pool:
+            results = pool.imap_unordered(func, morphologies, chunksize=100)
```

Comment:
```suggestion
            results = list(pool.imap_unordered(func, morphologies, chunksize=100))
```
Without the list the result is just a generator but when you try to iterate on it the Pool does not exist any more so it stays frozen.

Reply:
ah thanks, good catch!